### ISSUE TITLE
Failures during <Import Project...> parsing

### ISSUE THREAD

**Issue by spaun2002** [Caphyon/clang-power-tools#144]
From Clang Power Tools logs:

```
Sanitizing Z:\src\****.props
Test-Path : A parameter cannot be found that matches parameter name 'and'.
At line:1 char:52
+ (Test-Path("\build\native\Microsoft.Gsl.targets")  -and  !Exists("..\ ...
+                                                    ~~~~
    + CategoryInfo          : InvalidArgument: (:) [Test-Path], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Microsoft.PowerShell.Commands.TestPathCommand
```

In the props file the line is
`  <Import Project="$(PkgMicrosoft_Gsl)\build\native\Microsoft.Gsl.targets" Condition="Exists('$(PkgMicrosoft_Gsl)\build\native\Microsoft.Gsl.targets') AND !Exists('..\packages\Microsoft.Gsl.0.1.2.0\build\native\Microsoft.Gsl.target')"/>`

**Comment by hero101111** [comment]
Hello,

It seems we had issues with conditions/expressions that called `Exists()` more than once.
We've pushed a [fix ](https://github.com/Caphyon/clang-power-tools/commit/2627a6993fb6d2d9fb3208bc6a29d471e160ff45)for this.

Please retest, using the instructions below, and let us know if your project compiles correctly now.

Regards,
Gabriel

---

This fix is currently live only for the github repository.
The latest Visual Studio extension (2.1) does not contain the fix. It will be included in the next release.

Until then, as a workaround, you can manually replace `clang-build.ps1` in the VS extension directory with the latest from github. To locate the extension directory search for `clang-build.ps1` in `%LOCALAPPDATA%`

Don't forget to restart Visual Studio after replacing the script file.

**Comment by hero101111** [comment]
This issue was resolved in version `2.2` released on December 11th.
